flake8diff/vcs/git.py

```diff
@@ -65,3 +69,7 @@ def changed_files(self):
         return filter(self.filter_file,
                       iter(_execute(' '.join(command))
                            .splitlines()))
+
+    def _is_git_repository(self):
+        return _execute(
+            '{vcs} status'.format(vcs=self.vcs), strict=True, log_errors=False)
```

Comment:
@gregarmer I can agree and disagree here. My main concern is that `git status` will also check if the repository is consistent. I'll better keep the current implementation.